php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S4423.html

```diff
@@ -5,16 +5,15 @@ <h2>Noncompliant Code Example</h2>
 $ctx = stream_context_create([
   'ssl' =&gt; [
     'crypto_method' =&gt;
-      STREAM_CRYPTO_METHOD_TLSv1_1_CLIENT | // Noncompliant
-      STREAM_CRYPTO_METHOD_TLSv1_2_CLIENT,
+      STREAM_CRYPTO_METHOD_TLSv1_1_CLIENT // Noncompliant
   ],
 ]);
 </pre>
 <h2>Compliant Solution</h2>
 <pre>
 $ctx = stream_context_create([
     'ssl' =&gt; [
-        'crypto_method' =&gt; STREAM_CRYPTO_METHOD_SSLv3_CLIENT
+        'crypto_method' =&gt; STREAM_CRYPTO_METHOD_TLSv1_2_CLIENT
```

Comment:
the rule must be updated ?

Reply:
I don't think so, just some consistency to noncompliant/compliant is brought